libyul/AsmJsonConverter.cpp

```diff
@@ -37,12 +37,12 @@ Json AsmJsonConverter::operator()(Block const& _node) const
 	return ret;
 }
 
-Json AsmJsonConverter::operator()(TypedName const& _node) const
+Json AsmJsonConverter::operator()(NameWithDebugData const& _node) const
 {
 	yulAssert(!_node.name.empty(), "Invalid variable name.");
 	Json ret = createAstNode(originLocationOf(_node), nativeLocationOf(_node), "YulTypedName");
 	ret["name"] = _node.name.str();
-	ret["type"] = _node.type.str();
+	ret["type"] = "";
```

Comment:
Couldn't we just remove `type` completely? Maybe we also want to rename the name of the node `YulTypedName`? From my point of view It would be nice to get rid of this, but I guess the probability is high that this might break some tools.

Reply:
I didn't want to mess with the Json interface so I left it in and didnt rename anything :)